### PR TITLE
Update Catalan translation

### PR DESCRIPTION
**Pull request by duub**
#1582

### CODE REVIEW

**Comment by erciccione** [comment]
Ping @duub. This translation need a rebase and some fixes, please contact me so we ca work on this together

**Comment by duub** [comment]
Oh, sorry for not paying enough attention into this...

**Comment by duub** [comment]
@erciccione what I've to do to make a rebase?

**Comment by erciccione** [comment]
You can use the "resolve conflicts" button at the bottom of this Pull Request. More info: https://help.github.com/articles/resolving-a-merge-conflict-on-github/

**Comment by Monero-Pootle** [comment]
ping @duub

**Comment by erciccione** [comment]
I forgot to change user. @Monero-Pootle it's me with the pootle account

File: translations/monero-core_cat.ts [hunk 1]
```diff
@@ -2584,39 +2593,45 @@ For the case with Spend Proof, you don&apos;t need to specify the recipient addr
...
     </message>
     <message>
         <location filename="../main.qml" line="638"/>
         <source>
 
 WARNING: non default ring size, which may harm your privacy. Default of 7 is recommended.</source>
-        <translation type="unfinished"></translation>
+        <translation>
+
+ Avís: mida de l&apos;anell no predeterminada, la qual pot empitjorar la teva privacitat. Es recomana la configuració predeterminada de 7.</source>
```
[the remark this answers is not in the document]
Comment (duub) [reply translations/monero-core_cat.ts]: this one I don't understant whats wrong with it

**Comment by duub** [comment]
Hope i didn't miss anything... :P

File: translations/monero-core_cat.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (erciccione) [reply translations/monero-core_cat.ts]: There are two source tag. This is breaking the build. Please remove it

**Comment by erciccione** [comment]
@duub It needs multiple fixes. Please add me as collaborator to your repo, so i can fix them quickly. We need the repository clean from translations-related pull requests

**Comment by duub** [comment]
I think I've fixed the conflicts already

File: translations/monero-core_cat.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (erciccione) [reply translations/monero-core_cat.ts]: You need to remove the </source> at the end of the string or this line will keep breaking the build
`... predeterminada de 7.</source>`
at the end this section must look like:
```
 <translation>
 Avís: mida de l&apos;anell no predeterminada, la qual pot empitjorar la teva privacitat. Es recomana la configuració predeterminada de 7.
 </translation>
```

**Pull request merged**